### ISSUE TITLE
Multi Tenancy 

### ISSUE THREAD

**Issue by proverbial-ninja** [Jiro-Commerce/Jiro#2]
Is Multi tenancy in the road map??

**Comment by andrewmclagan** [comment]
Yes, although that would be further down the road perhaps 0.4.0 or later as there are allot of dependancies to implement. For the time-being i will tag this as 0.4.0
